pxd.c

```diff
@@ -409,6 +415,10 @@ static void pxd_request(struct fuse_req *req, uint32_t size, uint64_t off,
 	case REQ_WRITE | REQ_DISCARD:
 		pxd_discard_request(req, size, off, minor, flags, qfn);
 		break;
+	default:
+		printk(KERN_ERR"[%llu] REQ_OP_UNKNOWN(%#x): size=%d, off=%lld, minor=%d, flags=%#x\n",
+			req->in.h.unique, flags, size, off, minor, flags);
+		BUG();
```

Comment:
This should never be happening. This would be a kernel bug. A strong condition would ensure this gets the needed attention. Returning IO error may get thrown out as noise/bad drive or any other reasons.
Maybe once we gain sufficient confidence inhouse after testing, we can revisit to understand these conditions.

Reply:
If it's only for test, this needs to be a configurable option with no panic as default.

Reply:
revised.